File: array/217-Contains-Duplicate.js
/*
217. Contains Duplicate
leetcode.com/problems/contains-duplicate/
5 April 2023
David Pratt
*/

/**
 * @param {number[]} nums
 * @return {boolean}
 */
https: var containsDuplicate = function (nums) {
  let uniqueNums = new Set(nums);
  return uniqueNums.size !== nums.length;
};

File: array/217-Contains-Duplicate.py
/*
217. Contains Duplicate
https://leetcode.com/problems/contains-duplicate/
26 November 2023
David Pratt
*/


import numpy as np

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        unique_length = len( np.unique(nums) )
        return unique_length < len(nums)

        
